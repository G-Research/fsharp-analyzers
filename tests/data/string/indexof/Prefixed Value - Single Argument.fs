module A =
    let b = "b"

A.b.IndexOf("b")
