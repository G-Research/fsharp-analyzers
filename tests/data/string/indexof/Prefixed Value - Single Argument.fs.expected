GRA-STRING-003 | Warning | (6,0 - 6,16) | The usage of String.IndexOf with a single string argument is discouraged. Signal your intention explicitly by calling an overload.
